src/containers/Login/index.js

```diff
@@ -69,6 +74,17 @@ const LoginComponent = props => {
     return <SigningIn visible={visible} />;
   }
 
+  if (stage === 'profile') {
+    return (
+      <AddProfileDetails
+        visible={visible}
+        onClose={() => showLogin(false)}
+        form={form}
```

Comment:
a little confused here.

Reply:
Ok - I understand what you're doing. We should change/tweak this a little bit to make the logic simpler. Lets talk :)